### ISSUE TITLE
Pager - Axe Violation - Form elements must have labels

### ISSUE THREAD

**Issue by hegou33** [Sage/carbon#5284]
### Current behaviour

Pager component has an axe violation, ensure every form element has a label.
![image](https://user-images.githubusercontent.com/38658070/177157080-000ad253-e76c-4793-b42e-c67fd174689a.png)

### Expected behaviour

No accessibility violations should occur.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/pager--page-size-selection-options

### Carbon Version

109.1.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by Parsium** [comment]
FE-5233

**Comment by nicktitchmarsh** [comment]
#5282 relates to this issue which is a duplicate of FE-4659
